virtualizarr/tests/test_writers/test_icechunk.py

```diff
@@ -19,21 +19,16 @@
 from virtualizarr.zarr import ZArray
 
 if TYPE_CHECKING:
-    from icechunk import IcechunkStore, StorageConfig  # type: ignore[import-not-found]
+    from icechunk import IcechunkStore, Storage  # type: ignore[import-not-found]
 
 
 @pytest.fixture(scope="function")
 def icechunk_filestore(tmpdir) -> "IcechunkStore":
-    from icechunk import IcechunkStore, StorageConfig
+    from icechunk import Repository, Storage
 
-    storage = StorageConfig.filesystem(str(tmpdir))
-
-    # TODO if icechunk exposed a synchronous version of .open then we wouldn't need to use asyncio.run here
-    # TODO is this the correct mode to use?
-    store = IcechunkStore.create(storage=storage, mode="w")
-
-    # TODO instead yield store then store.close() ??
-    return store
```

Comment:
Should we close the session when tearing down the fixture? Or does it not matter?

Reply:
Does not matter at this time